File: generated-code/codecraft/scala/TcpReadWrite/src/main/scala/Runner.scala
import java.io._
import java.net.Socket
import util.StreamUtil

object Runner extends App {
    if (args.size != 2) {
        throw new Exception("Pass host and port as parameters")
    }
    val host = args(0)
    val port = Integer.parseInt(args(1))

    val socket = new Socket(host, port)
    socket.setTcpNoDelay(true)

    val inputStream: InputStream = new BufferedInputStream(socket.getInputStream())
    val input: model.PlayerView = model.PlayerView.readFrom(inputStream)

    println(input)

    val outputStream: OutputStream = new BufferedOutputStream(socket.getOutputStream())
    input.writeTo(outputStream)
    outputStream.flush()

    socket.close()
}

File: generated-code/java/example/FileReadWrite/src/main/java/trans_gen_test/Runner.java
import java.io.*;

import trans_gen_test.util.StreamUtil;

public class Runner {
    public static void main(String[] args) throws IOException {
        String inputFile = args[0];
        String outputFile = args[1];
        int repeat = Integer.parseInt(args[2]);

        for (int i = 0; i < repeat; i++) {
            InputStream inputStream = new BufferedInputStream(new FileInputStream(inputFile));
            trans_gen_test.Example input = trans_gen_test.Example.readFrom(inputStream);
            if (repeat == 1) {
                System.out.println(input);
            }
            OutputStream outputStream = new BufferedOutputStream(new FileOutputStream(outputFile));
            input.writeTo(outputStream);
            outputStream.flush();
            outputStream.close();
        }
    }
}